### PR TITLE
toggle for enable/disable stats

### PR DESCRIPTION
**Pull request by dlayton66**

### CODE REVIEW

File: discord_bots/commands.py [hunk 1]
```diff
@@ -2127,6 +2127,19 @@ async def disableleaderboard(ctx: Context):
...
+async def disablestats(ctx: Context):
+    session = Session()
+    player = session.query(Player).filter(Player.id == ctx.message.author.id).first()
+    player.stats_enabled = False
+    session.commit()
+    await send_message(
+        ctx.message.channel,
+        embed_description="!stats disabled",
+        colour=Colour.blue()
+    )
```
Comment (dwayneyuen) [inline discord_bots/commands.py:2140]:
```suggestion
    )
    session.close()
```
this releases the session faster on the db

File: discord_bots/commands.py [hunk 2]
```diff
@@ -2205,6 +2218,19 @@ async def enableleaderboard(ctx: Context):
...
+async def enablestats(ctx: Context):
+    session = Session()
+    player = session.query(Player).filter(Player.id == ctx.message.author.id).first()
+    player.stats_enabled = True
+    session.commit()
+    await send_message(
+        ctx.message.channel,
+        embed_description="!stats enabled",
+        colour=Colour.blue()
+    )
```
Comment (dwayneyuen) [inline discord_bots/commands.py:2231]:
```suggestion
    )
    session.close()
```

File: discord_bots/commands.py [hunk 3]
```diff
@@ -3986,160 +4012,167 @@ def win_rate(wins, losses, ties):
...
-    DEFAULT_TRUESKILL_MU = float(os.getenv("DEFAULT_TRUESKILL_MU") or default_rating.mu)
-
-    # Filter players that haven't played a game
-    players = list(
-        filter(
-            lambda x: x.rated_trueskill_mu != default_rating.mu
-            and x.rated_trueskill_mu != DEFAULT_TRUESKILL_MU,
-            players,
+    if not player.stats_enabled:
+        channel_output = "You have disabled !stats"
```
Comment (dwayneyuen) [inline discord_bots/commands.py:4019]: better here i think is to send the channel message and return early, then the rest of the function is unchanged. so like 

```
    session.close()
    if ctx.message.guild:
        member_: Member | None = ctx.message.guild.get_member(player.id)
        await send_message(
            ctx.message.channel,
            embed_description=channel_output,
            colour=Colour.blue(),
        )
        return
```

**Review by dwayneyuen — COMMENTED** [review]
sick!!! minor requests

File: discord_bots/commands.py (hunk 3 above)
Comment (dlayton66) [reply discord_bots/commands.py:4021]: yeah, i'm always trying to DRY as much as possible, but having such a huge else block felt bad too

**Review by dwayneyuen — APPROVED** [review]
SHIP IT

**Pull request merged**